component-samples/aio/src/main/java/org/fidoalliance/fdo/sample/AioServerApp.java

```diff
@@ -0,0 +1,279 @@
+    ctx.addParameter(AioAppSettings.MANUFACTURER_KEYSTORE_PWD,
+        AioConfigLoader.loadConfig(AioAppSettings.MANUFACTURER_KEYSTORE_PWD));
+    ctx.addParameter(AioAppSettings.MANUFACTURER_KEYSTORE_TYPE,
+        AioConfigLoader.loadConfig(AioAppSettings.MANUFACTURER_KEYSTORE_TYPE));
+
+    ctx.addParameter(AioAppSettings.TO0_SCHEDULING_ENABLED,
+        AioConfigLoader.loadConfig(AioAppSettings.TO0_SCHEDULING_ENABLED));
+    ctx.addParameter(AioAppSettings.TO0_SCHEDULING_INTREVAL,
+        AioConfigLoader.loadConfig(AioAppSettings.TO0_SCHEDULING_INTREVAL));
+    ctx.addParameter(AioAppSettings.TO0_MAX_WS,
+        AioConfigLoader.loadConfig(AioAppSettings.TO0_MAX_WS));
+
+
+
+
+
+    ctx.addApplicationListener(DbStarter.class.getName());
+    ctx.addApplicationListener(AioContextListener.class.getName());
+    ctx.setParentClassLoader(ctx.getClass().getClassLoader());
+
+    //load owner protocol servlet and api
+    Wrapper wrapper = tomcat.addServlet(ctx, "fdoServlet", new ProtocolServlet());
+
+    wrapper.addMapping(getMessagePath(Const.TO2_HELLO_DEVICE));
+    wrapper.addMapping(getMessagePath(Const.TO2_GET_OVNEXT_ENTRY));
+    wrapper.addMapping(getMessagePath(Const.TO2_PROVE_DEVICE));
+    wrapper.addMapping(getMessagePath(Const.TO2_DEVICE_SERVICE_INFO_READY));
+    wrapper.addMapping(getMessagePath(Const.TO2_DEVICE_SERVICE_INFO));
+    wrapper.addMapping(getMessagePath(Const.TO2_DONE));
+
+    wrapper.addMapping(getMessagePath(Const.TO0_HELLO));
+    wrapper.addMapping(getMessagePath(Const.TO0_OWNER_SIGN));
+    wrapper.addMapping(getMessagePath(Const.TO1_HELLO_RV));
+    wrapper.addMapping(getMessagePath(Const.TO1_PROVE_TO_RV));
+
+    wrapper.addMapping(getMessagePath(Const.DI_APP_START));
+    wrapper.addMapping(getMessagePath(Const.DI_SET_HMAC));
+    wrapper.setAsyncSupported(true);
+
+    wrapper = tomcat.addServlet(ctx, "voucherServlet",
+        new OwnerVoucherServlet());
+    wrapper.setAsyncSupported(true);
+    wrapper.addMapping("/api/v1/owner/vouchers/*");
+    wrapper = tomcat.addServlet(ctx, "replacementVoucherServlet",
+        new OwnerReplacementVoucherServlet());
+    wrapper.addMapping("/api/v1/owner/newvoucher/*");
+    wrapper = tomcat.addServlet(ctx, "setupinfoServlet",
+        new OwnerSetupInfoServlet());
+    wrapper.addMapping("/api/v1/owner/setupinfo/*");
+    wrapper.setAsyncSupported(true);
+    wrapper = tomcat.addServlet(ctx, "ownerCustomerServlet",
+        new OwnerCustomerServlet());
+    wrapper.addMapping("/api/v1/owner/customer/*");
+
+    wrapper = tomcat.addServlet(ctx, "SystemResource", new OwnerSystemResourceServlet());
+    wrapper.setAsyncSupported(true);
+    wrapper.addMapping("/api/v1/device/svi");
+
+    wrapper = tomcat.addServlet(ctx, "sviSettingsServlet",
+        new OwnerSviSettingsServlet());
+    wrapper.addMapping("/api/v1/owner/svi/settings/*");
+
+    wrapper = tomcat.addServlet(ctx, "DiApi", new DiApiServlet());
+    wrapper.addMapping("/api/v1/vouchers/*");
```

Comment:
Some API mappings are missing
Eg: /api/v1/assign/?id= ; POST /api/v1/customers/?id=<customer_id>&name=<customer_name>

Reply:
all api's  should be covered by /api/v1/*    all servlet api patterns are "/api/v1/*" e.g. /api/v1/assign/*""